sktime/forecasting/compose/tests/test_reduce.py

```diff
@@ -381,3 +381,110 @@ def test_multioutput_direct_equivalence_tabular_linear_regression(fh):
     np.testing.assert_array_almost_equal(
         y_pred_direct.to_numpy(), y_pred_multioutput.to_numpy()
     )
+
+
+# this expected value created by Lovkush Agarwal by running code locally in Mar 2021
+EXPECTED_AIRLINE_LINEAR_RECURSIVE = [
+    397.28122475088117,
+    391.0055770755232,
+    382.85931770491493,
+    376.75382498759643,
+    421.3439733242519,
+    483.7127665080476,
+    506.5011555360703,
+    485.95155173523494,
+    414.41328025499604,
+    371.2843322707713,
+    379.5680077722808,
+    406.146827316167,
+    426.48249271837176,
+    415.5337957767289,
+    405.48715913377714,
+    423.97150765765025,
+    472.10998764155966,
+    517.7763038626333,
+    515.6077989417864,
+    475.8615207069196,
+    432.47049089698646,
+    417.62468250043514,
+    435.3174101071012,
+    453.8693707695759,
+]
+
+# this expected value created by Lovkush Agarwal by running code locally in Mar 2021
+EXPECTED_AIRLINE_LINEAR_DIRECT = [
+    388.7894742436609,
+    385.4311737990922,
+    404.66760376792183,
+    389.3921653574014,
+    413.5415037170552,
+    491.27471550855756,
+    560.5985060880608,
+    564.1354313250545,
+    462.8049467298484,
+    396.8247623180332,
+    352.5416937680942,
+    369.3915756974357,
+    430.12889943026323,
+    417.13419789042484,
+    434.8091175980315,
+    415.33997516059355,
+    446.97711875155846,
+    539.6761098618977,
+    619.7204673400846,
+    624.3153932803112,
+    499.686252475341,
+    422.0658526180952,
+    373.3847171492921,
+    388.8020135264563,
+]
+
+
+@pytest.mark.parametrize(
+    "forecaster, expected",
+    [
+        (
+            DirectTabularRegressionForecaster(LinearRegression()),
+            EXPECTED_AIRLINE_LINEAR_DIRECT,
+        ),
+        (
+            MultioutputTabularRegressionForecaster(LinearRegression()),
+            EXPECTED_AIRLINE_LINEAR_DIRECT,
+        ),
+        (
+            RecursiveTabularRegressionForecaster(LinearRegression()),
+            EXPECTED_AIRLINE_LINEAR_RECURSIVE,
+        ),
+        (
+            DirectTimeSeriesRegressionForecaster(
+                make_pipeline(Tabularizer(), LinearRegression())
+            ),
+            EXPECTED_AIRLINE_LINEAR_DIRECT,
+        ),
+        (
+            MultioutputTimeSeriesRegressionForecaster(
+                make_pipeline(Tabularizer(), LinearRegression())
+            ),
+            EXPECTED_AIRLINE_LINEAR_DIRECT,
```

Comment:
Since multioutput and direct are equivalent for linear regression, we could remove them. There is already a test that checks their equivalence. If you want to leave them in, I would add a comment stating that they are expected to be equivalent. Otherwise it may be confusing to see that we check multioutput against results for direct.

Reply:
I have no strong thought either way; I added some comments.

Reply:
I suspect these tests are reasonably fast, so let's leave them in.